rest/routes/user.py

```diff
@@ -61,9 +67,7 @@ def register():
 
```

Comment:
I think password hashing should take place in models/user.py. I would also add verify_password function there so that handling password elsewhere is easier. No need to hash it outside models/user.py.